packages/core/src/components/organisms/HeaderOptionsMenu/HeaderOptionsMenu.js

```diff
@@ -141,6 +157,14 @@ const HeaderOptionsMenu = ({
     <View style={styles.container}>
       {HeaderItemList}
       {menuActions.length !== 0 && <DropdownMenu>{MenuItemList}</DropdownMenu>}
+      {showPrintTemplateSelector && (
+        <PopupPrintTemplate
+          visible={showPrintTemplateSelector}
+          onClose={closePrintTemplateSelector}
+          model={model}
+          modelId={modelId}
+        />
+      )}
```

Comment:
This part of code is exactly the same as above, put them in a function to avoid duplicate code